EIPS/eip-draft_backward_compat_PQ_migration.md

```diff
@@ -0,0 +1,198 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7693
```

Reply:
Updated to use assigned EIP number. Thanks!